docs/guides/callbacks.md

```diff
@@ -0,0 +1,58 @@
+# Mocking Callbacks with Prism
+
+## What are callbacks?
```

Comment:
New lines between all the paragraphs — or if you're lazy you can just run prettier on it:

`npx prettier --write ./docs/guides/callbacks.md`

I did that for you in #772

Reply:
Resolved via merging #772